.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Run tests
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '3 20 * * SUN'
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Are we going to replace the Windows test too? We will want to build from the project files there so a separate build script probably makes sense.

Reply:
Yeah, I started with these two. I'm starting the Windows one now!